test/Enum.rb

```diff
@@ -196,4 +196,14 @@ def test_using_virtual_pixel_method_does_not_cause_endless_loop
       assert_equal(value, img.virtual_pixel_method)
     end
   end
+
+  def test_storage_type_name
+    img = Magick::Image.new(20, 20)
+    pixels = img.export_pixels(0, 0, 20, 20, 'RGB').pack('D*')
+
+    exception = assert_raise(ArgumentError) do
```

Comment:
Should we do `assert_raise_with_message` here?

Reply:
nice 👍

Reply:
Hmm, unfortunately, https://github.com/test-unit/test-unit gem does not support `assert_raise_with_message`.

Reply:
Changed to `assert_raise_message` because it is important things what I want to check the string of exception message.